packages/cf.js/src/provider.ts

```diff
@@ -287,14 +287,12 @@ export class Provider {
     return this.appInstances[id];
   }
 
-  /**
-   * @ignore
-   */
-  private validateEventType(eventType: EventType) {
-    // if (!this.validEventTypes.includes(eventType)) {
-    //   throw new Error(`"${eventType}" is not a valid event`);
-    // }
-  }
+  // FIXME: Either implement this or remove it
```

Comment:
I'll defer to @joelalejandro on this as there's an issue to properly fix how messages and events are handled